src/solver/conv_asm_implicit_gemm_gtc_fwd.cpp

```diff
@@ -517,7 +504,11 @@ bool ConvAsmImplicitGemmGTCDynamicFwdXdlops::IsApplicable(const ConvolutionConte
     if(ctx.group_counts != 1)
         return false;
 
-    return FindImplicitGemmGtcDynamicFwdKernel(ctx, nullptr, nullptr, nullptr);
+    bool isValid;
+    std::tie(isValid, std::ignore, std::ignore, std::ignore) =
+        FindImplicitGemmGtcDynamicFwdKernel(ctx);
```

Comment:
[Q] Is it clear for you that now `FindImplicitGemmGtcDynamicFwdKernel()` does more useless job than before?

Reply:
Actually, in function ConvAsmImplicitGemmGTCDynamicFwdXdlops::IsApplicable, we need the first element in tuple, but in function ConvAsmImplicitGemmGTCDynamicFwdXdlops::GetSolution, we need the others in tuple. The functionality of FindImplicitGemmGtcDynamicFwdKernel() is same as the previous one.

Reply:
Of course. I just would like to make it clear for you that in `ConvAsmImplicitGemmGTCDynamicFwdXdlops::IsApplicable`, the function does more useless job than before. I would appreciate if you optimize this in the future.